src/main.rs

```diff
@@ -35,14 +49,20 @@ const ONION_MESSAGES_OPTIONAL: u32 = 39;
 async fn main() -> Result<(), ()> {
     simple_logger::init_with_level(log::Level::Info).unwrap();
 
-    let args = match parse_args() {
-        Ok(args) => args,
-        Err(e) => {
-            error!("Invalid arguments: {e}");
-            return Err(());
-        }
-    };
+    let lnd_config = Config::including_optional_config_files(empty::<&str>())
+        .unwrap_or_exit()
+        .0;
 
+    let mut cert_str = String::new();
+    if let Some(path_str) = lnd_config.cert.to_str() {
+        cert_str.push_str(path_str);
+    }
+    let mut macaroon_str = String::new();
+    if let Some(path_str) = lnd_config.macaroon.to_str() {
+        macaroon_str.push_str(path_str);
+    }
```

Comment:
Assuming that our config parsing enforced the presence of macaroon/cert, can we shorten these to something like:
`let macaroon_str = lnd_config.macaroon.to_str().unwrap().to_string();`

Reply:
Actually, `to_str` is incorrect because it tries to convert `Path` into `str` which may fail. This whole section of code shouldn't be needed, just call `LndCfg::new(lnd_config.address, lnd_config.cert, lnd_config.macaroon)` below.

Reply:
yes I agree. thanks @carlaKC @Kixunil, I will update the commit.

But then I have to update the  `LndCfg` struct data type for cert and macaroon to PathBuf.